### ISSUE TITLE
Please remove empty lines between features

### ISSUE THREAD

**Issue by msrd0** [slint-ui/document-features#17]
When I run `cargo expand`, this is the generated output from the macro:

```markdown
* **`openapi`** —  Add `OpenapiType` implementations for `AATreeSet` and `AATreeMap`.

* **`serde`** —  Add `(De)Serialize` implementations for `AATreeSet` and `AATreeMap`.
```

Unfortunately, this triggers some markdown renderers (including [`pulldown-cmark`](https://crates.io/crates/pulldown-cmark) and [`grip`](https://github.com/joeyespo/grip)) to view them as two separate lists with one item each, which means they are rendered with too much spacing between the two features. If you removed the empty line, they would be treated as one list with two items, and be rendered with less space, just like rustdoc.

**Comment by ogoffart** [comment]
I wonder if the change would break existing crate and not look good on docs.rs if the documentation uses newlines or paragraph within their feature docs.

I made a quick try and it seems to work alright. So I guess we can make the change without breaking compatibility.
